### PR TITLE
Restructure repository and use setuptools.

### PR DESCRIPTION
**Pull request by gmierz**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    keywords = "Android power and battery usage",
+    url = "https://github.com/mozilla/powerusage-android",
+    packages=[
+        'utils',
+        'tests',
+    ],
+    classifiers=[
+        "Development Status :: Beta",
+    ]
+)
```
Comment (stephendonner) [inline setup.py]: Nit: newline

File: tests/android_os_baseline_by_percent.py [hunk 2]
```diff
@@ -13,7 +13,7 @@
     parse_battery_info,
     wait_for_drop,
 )
-from utils import finish_same_line, write_same_line
+from utils.utils import finish_same_line, write_same_line
 
 OUTPUT = "/home/sparky/Documents/mozwork/"
```
Comment (stephendonner) [inline tests/android_os_baseline_by_percent.py]: Ditto for removing/refactoring/rebasing this out.

File: tests/android_os_baseline_test.py [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
-from android_parser import AndroidParser
-from data_saver import DataSaver
-from adb_utils import get_phone_model, get_battery_info, parse_battery_info
-from utils import finish_same_line, write_same_line
+from utils.android_parser import AndroidParser
+from utils.data_saver import DataSaver
+from utils.adb_utils import get_phone_model, get_battery_info, parse_battery_info
+from utils.utils import finish_same_line, write_same_line
 
 OUTPUT = "/home/sparky/Documents/mozwork/"
```
Comment (stephendonner) [inline tests/android_os_baseline_test.py]: o hai; this too

**Comment by gmierz** [comment]
Yup, we can abstract the RESULTSDIR in another PR.

**Comment by stephendonner** [comment]
Make-It-So

**Pull request merged**